flamedisx/source.py

```diff
@@ -47,6 +52,10 @@ class Source:
     #: for use in domain / cross-domain
     final_dimensions: ty.Tuple[str] = tuple()
 
+    #: Names of penulitmate model dimensions;
```

Comment:
OK, though in the future we could autodetect these, or setup a more general system of specifying default max dimsizes per block.

Reply:
Agree